packages/dds/legacy-dds/src/signal/sharedSignal.ts

```diff
@@ -163,6 +163,11 @@ export class SharedSignalClass<T extends SerializableTypeForSharedSignal = any>
 		this.emit("notify", op.metadata, isLocal);
 	}
 
+	// Nothing to roll back. Allowing other DDSs to handle rollback if necessary.
```

Comment:
"DDSes" is our official plural.
```suggestion
	// Nothing to roll back. Allowing other DDSes to handle rollback if necessary.
```